vulture_os/system/tenants/urls.py

```diff
@@ -43,7 +43,7 @@
             views.tenants_edit,
             name="system.tenants.edit"),
 
-    re_path('api/v1/system/tenants/(?P<object_id>[A-Fa-f0-9]+)?$', api.TenantsAPIv1.as_view(), name="system.tenants.api"),
-    re_path('api/v1/system/tenants/(?P<object_id>[A-Fa-f0-9]+)?/$', api.TenantsAPIv1.as_view(), name="system.tenants.api"),
+    path('api/v1/system/tenants/', api.TenantsAPIv1.as_view(), name="system.tenants.api"),
+    path('api/v1/system/tenants/<str:object_id>/', api.TenantsAPIv1.as_view(), name="system.tenants.api"),
```

Comment:
Please reuse the given regex
```suggestion
    path('api/v1/system/tenants/', api.TenantsAPIv1.as_view(), name="system.tenants.api"),
    re_path('api/v1/system/tenants/(?P<object_id>[A-Fa-f0-9]+)?/$', api.TenantsAPIv1.as_view(), name="system.tenants.api"),
```

Reply:
resolved in b72d23a